src/toil_scripts/gatk_processing/gatk_preprocessing.py

```diff
@@ -269,6 +269,36 @@ def docker_call_preprocess(work_dir, tool_parameters, tool, java_opts=None,
     outfile: file           Filehandle that stderr will be passed to
     sudo: bool              If the user wants the docker command executed as sudo
     """
+
+    for filename in inputs:
```

Comment:
Maybe this is the time to start consolidating all these `docker_call` instances.